docs/docs/guides/web3_migration_guide/subscribe_migration_guide.md

```diff
@@ -30,7 +69,18 @@ web3.eth.clearSubscriptions(function (error, success) {
 
 // in 4.x
 const subscription = await web3.eth.subscribe('newHeads');
-subscription.unsubscribe().then(
-	console.log(), // [...] Array of subsription ids
+
+// note that in version 4.x the way you get notified for `data` and `error` has changed
+newBlocksSubscription.on('data', async blockhead => {
+	console.log('New block header: ', blockhead);
+});
+newBlocksSubscription.on('error', error =>
+	console.log('Error when subscribing to New block header: ', error),
 );
+
+const ids = await web3.eth.clearSubscriptions(function (error, success);
```

Comment:
seems its not correct snippet, also we dnt support passing function in it.;

Reply:
Thanks for noting that. I missed stripping out the parameters of `clearSubscriptions`. It has been corrected now.